### PR TITLE
update harvesting pages

### PR DESCRIPTION
**Pull request by NemTechCN**

### CODE REVIEW

**Comment by decentraliser** [comment]
image
i18n key

**Comment by decentraliser** [comment]
image
cursor: pointer

File: src/components/forms/persistent-delegation-request/PersistentDelegationRequest.vue [hunk 1]
```diff
@@ -1,60 +1,29 @@
...
-      <div class="new_model_btn">
-        <span class="modal_btn pointer radius" @click="submit">{{$t('confirm')}}</span>
-      </div>
-    </form>
-  </Modal>
+  <div>
+    <div>
+      <input v-model="nodeUrl" class="netWork-input" v-focus type="text"
+             :placeholder="$t('Please_enter_the_domain_name_or_IP_address_of_the_node_network')"/>
+      <span @click="searchNodeInfo" class="select_button">{{$t('Search')}}</span>
```
Comment (decentraliser) [inline src/components/forms/persistent-delegation-request/PersistentDelegationRequest.vue]: this input should be validated by vee-validate

**Comment by decentraliser** [comment]
image

The alignment of the buttons and the upper part feel a bit awkward

**Comment by decentraliser** [comment]
image

I think that Type: transfer would be more appropriate here

**Comment by decentraliser** [comment]
image
Should change "Next step" to something else, like "close"

**Comment by decentraliser** [comment]
image

1. Typo on "Desactivate" 
2. There is no desactivate function is it? If not, remove this button

1.

File: src/components/forms/persistent-delegation-request/PersistentDelegationRequest.vue (hunk 1 above)
Comment (NemTechCN) [reply src/components/forms/persistent-delegation-request/PersistentDelegationRequest.vue]: fixed

**Comment by NemTechCN** [comment]
> image
> 
> 1. Typo on "Desactivate"
> 2. There is no desactivate function is it? If not, remove this button

fixed

File: src/components/forms/account-link/AccountLinkTs.ts [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+    get password() {
+        return this.app.loadingOverlay.temporaryInfo.password
+    }
+
+    get remoteCreatedPublicKey() {
+        return new AppWallet(this.wallet).getRemoteAccount(this.password).publicKey
+    }
+
+    async signAndAnnounce() {
```
Comment (evias) [inline src/components/forms/account-link/AccountLinkTs.ts]: needs rebase with other PRs

**Comment by decentraliser** [comment]
Need to add some logic when getting the node public key:

“role” field differs 1 for api, 2 for peers, 3 for dual; SDK should have mapping of that

harvesting does not work with role 1

**Pull request closed without merging**